Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM buildpack-deps:xenial
 
 ENV GO_VERSION 1.8
-ENV KUBECTL_VERSION 1.5.3
+ENV KUBECTL_VERSION 1.6.0
```

Comment:
For Dockerfile, I think we can just use latest. This is just for the kubectl that is available in devenv as a convenience and it's backward compatible anyway. Parameterizing the Dockerfile would require the user to pass some indicator when running `./devenv.sh` and I'm just not sure it's needed.

Agree, of course, with the constant-izing elsewhere.